src/main.py

```
@@ -35,68 +35,75 @@
         # Dedupe feeds
         dupes = manager.dedupe_feeds()
         if dupes:
             console.print(f"[yellow]Removed {dupes} duplicate feeds[/yellow]")
-        
+
         # Process feeds
         with Progress() as progress:
-            task = progress.add_task("[cyan]Analyzing feeds...", total=len(manager.feeds))
-            
+            task = progress.add_task(
+                "[cyan]Analyzing feeds...", total=len(manager.feeds)
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)